client/components/reader-infinite-stream/row-renderers.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External Dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import ConnectedSubscriptionListItem from 'blocks/reader-subscription-list-item/connected';
+
+export const siteRowRenderer = ( {
```

Comment:
this is a bit weird still. should be items instead of sites and infinite-stream shouldn't need to know about followSource etc.